md/tests/test_views.py

```diff
@@ -0,0 +1,61 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from md.models import Agency, Stop
+
+
+class ViewTests(TestCase):
+    multi_db = True
+
+    def test_home(self):
+        response = self.client.get(reverse('md:home'))
+        self.assertEqual(200, response.status_code)
+
+    def test_agency_detail(self):
+        agency = Agency.objects.create(name="Maryland State Police")
```

Comment:
better to use the Agency factory in these tests, so that if/when additional required attributes are added to Agency this code wouldn't need to be changed (because the Agency factory would auto-generate the new attributes)